CONTRIBUTING.md

```diff
@@ -1,40 +1,143 @@
-* Run `make check-before-commit` before submitting the PR.
-  This will require you to either install or be in developer mode with: `python setup.py install/develop`.
-* In general, do not catch exceptions without good reason.
-  * catching non-fatal exceptions.
-    Log the exception as a warning.
-  * adding more information about what may have caused the error.
-    Raise a new exception using ``raise_from(NewException("message"), oldException)`` from ``future``.
-    Do not log this, as it creates redundant/confusing logs.
-* **If you are new to the project don't forget to add your name and affiliation to the `.zenodo.json` file.**
-
-## Contributing issues
-
-When opening a new Issue, please take the following steps:
-
-1. Search GitHub and/or [Neurostars](http://neurostars.org) for your issue to avoid duplicate reports.
-   Keyword searches for your error messages are most helpful.
-2. If possible, try updating to master and reproducing your issue,
-   because we may have already fixed it.
-3. OS and version
-4. Nipype version
-5. Output of: `import nipype; nipype.get_info()`
-6. Versions of underlying tools (e.g., ANTS, FSL, SPM, etc.,.)
-7. Any script, or output log, in a gist (gist.github.com)
-8. When applicable, and where possible, pointers to relevant data files.
+# Contributing to Nipype
+
+Welcome to the Nipype repository! We're excited you're here and want to contribute.  
+
+These guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
+
+Before you start you'll need to set up a free [GitHub][link_github] account and sign in. Here are some [instructions][link_signupinstructions].
+
+Already know what you're looking for in this guide? Jump to the following sections:
+* [Understanding issue labels](#issue-labels)
+* [Making a change](#making-a-change)
+* [How to tag pull requests](#tagging-pull-requests)
+* [Notes for new code](#notes-for-new-code)
+* [Recognizing contributions](#recognizing-contributions)
+
+## Issue labels
+
+The current list of issue labels are [here][link_labels] and include:
+
+* [![Bugs](https://img.shields.io/badge/-bugs-fc2929.svg)][link_bugs] *These issues point to problems in the project.*
+
+    If you find new a bug, please provide as much information as possible to recreate the error.
+    The [issue template][link_issue_template] will automatically populate any new issue you open, and contains information we've found to be helpful in addressing bug reports.
+    Please fill it out to the best of your ability!
+
+    If you experience the same bug as one already listed in an open issue, please add any additional information that you have as a comment.
+
+* [![Help Wanted](https://img.shields.io/badge/-help%20wanted-c2e0c6.svg)][link_helpwanted] *These issues contain a task that a member of the team has determined we need additional help with.*
```

Comment:
This links to "help wanted", not "help-wanted".

Reply:
🤦‍♀️  Patched with b9a38d1 !